tests/mock_tests/aclorch_ut.cpp

```diff
@@ -307,18 +307,18 @@ namespace aclorch_test
             ASSERT_EQ(gIntfsOrch, nullptr);
             gIntfsOrch = new IntfsOrch(m_app_db.get(), APP_INTF_TABLE_NAME, gVrfOrch);
 
-            ASSERT_EQ(gNeighOrch, nullptr);
-            gNeighOrch = new NeighOrch(m_app_db.get(), APP_NEIGH_TABLE_NAME, gIntfsOrch);
-
-            ASSERT_EQ(gRouteOrch, nullptr);
-            gRouteOrch = new RouteOrch(m_app_db.get(), APP_ROUTE_TABLE_NAME, gNeighOrch, gIntfsOrch, gVrfOrch);
-
             TableConnector applDbFdb(m_app_db.get(), APP_FDB_TABLE_NAME);
             TableConnector stateDbFdb(m_state_db.get(), STATE_FDB_TABLE_NAME);
 
             ASSERT_EQ(gFdbOrch, nullptr);
             gFdbOrch = new FdbOrch(applDbFdb, stateDbFdb, gPortsOrch);
 
+            ASSERT_EQ(gNeighOrch, nullptr);
```

Comment:
Please include the ut test results in the PR.

Reply:
> A few comments:
> 1, The ARP flush should not be tied to FDB remove/flush event as they are separate. For our use case, we should flush the ARP entry which had the macs on the port that was deleted.
> 2, We should add vs-tests to cover function supported by the code changes

Could you please elaborate on why we should NOT remove ARP entries that have the same MAC as the one we flushed.

Reply:
Is it because, everytime, a MAC age out happens, we end up flushing ARP entry as well? If so, isnt that better?

Reply:
No, not necessarily good.  MAC and ARP are independent. they have different timers. If MAC is aged, the neighbor could still be valid and thus the routes could still point to it. 

BTW: when you have routes pointing to the neighbor, and if you just flush the ARP for that neighbor, what is going to happen in the kernel?